lib/cluster/membership.c

```diff
@@ -597,6 +597,29 @@ crm_set_status_callback(void (*dispatch) (enum crm_status_type, crm_node_t *, co
     peer_status_callback = dispatch;
 }
 
+/*!
+ * \internal
+ * \brief Tell the library whether to automatically reap lost nodes
+ *
+ * If \c true (the default), calling \c crm_update_peer_proc() will also update
+ * the peer state to \c CRM_NODE_MEMBER or \c CRM_NODE_LOST, and
+ * \c pcmk__update_peer_state() will reap peers whose state changes to anything
+ * other than \c CRM_NODE_MEMBER.
+ *
+ * Callers should leave this enabled unless they plan to manage the cache
+ * separately on their own.
+ *
+ * \param[in] enable  \c true to enable automatic reaping, \c false to disable
+ */
+void
+pcmk__cluster_set_autoreap(bool enable)
+{
+    /* @TODO Review and update comment (at the very least, it doesn't reflect
+     * \c pcmk__reap_unseen_nodes())
```

Comment:
It kinda does -- `pcmk__reap_unseen_nodes()` calls `update_peer_state_iter()` which is essentially identical to `pcmk__update_peer_state()`. You could replace `pcmk__update_peer_state()` with "updating the peer state".